src/main/java/com/example/springsql/entities/Student.java

```diff
@@ -1,14 +1,28 @@
 package com.example.springsql.entities;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.Collection;
 import java.util.Objects;
 
+@Entity
 public class Student {
+    @Id
+    @GeneratedValue
     private long id;
     private String name;
     private int age;
+    @JsonIgnore
+    @ManyToOne
+    private Faculty faculties;
```

Comment:
```suggestion
    @ManyToOne
    @JoinColumn(name = "faculty_id")
    private Faculty faculty;
```
JsonIgnore нужно убрать, иначе не сможем задать факультет студенту
Аннотацию JoinColumn переносим сюда и меняем имя колонки
Также поменяем имя поля, чтобы mappedBy работало.